src/audio/pipeline/pipeline-graph.c

```diff
@@ -260,6 +261,11 @@ static int pipeline_comp_complete(struct comp_dev *current,
 
 	/* complete component init */
 	current->pipeline = ppl_data->p;
+
+	if (comp_is_llext(current))
+		/* pipelines are allocated using rzalloc(), so initially .init_time = 0 */
+		current->pipeline->init_time = sof_cycle_get_64();
+
```

Comment:
Not following why this is different for llext?

UPDATE: now  saa the new Kconfig for min lifetime, but maybe a comment would be good here as well.